lib/wrappers/openssl.nim

```diff
@@ -475,11 +475,11 @@ proc SSL_accept*(ssl: SslPtr): cint{.cdecl, dynlib: DLLSSLName, importc.}
 proc SSL_pending*(ssl: SslPtr): cint{.cdecl, dynlib: DLLSSLName, importc.}
 
 proc BIO_new_mem_buf*(data: pointer, len: cint): BIO{.cdecl,
-    dynlib: DLLSSLName, importc.}
+    dynlib: DLLUtilName, importc.}
 proc BIO_new_ssl_connect*(ctx: SslCtx): BIO{.cdecl,
```

Comment:
DLLSSLName or DLLUtilName?

Reply:
Sure is DLLUtilName

Reply:
then can you change it? (I'm referring to `BIO_new_ssl_connect`)

Reply:
'BIO_new_ssl_connect' is in DLLSSLName

Reply:
indeed, looks like it. What about this:

pkg-config --libs libcrypto
-L/usr/local/Cellar/openssl@1.1/1.1.1g/lib -lcrypto

nm /usr/local/Cellar/openssl@1.1/1.1.1g/lib/libcrypto.1.1.dylib | grep OPENSSL_sk_num    0000000000127ffc T _OPENSSL_sk_num
nm /usr/local/Cellar/openssl@1.1/1.1.1g/lib/libcrypto.1.1.dylib | grep OPENSSL_config
000000000005232b T _OPENSSL_config

=> looks like `OPENSSL_sk_num` should also be changed in this PR

I'm wondering if there's a better way than using `nm` ?

Reply:
In fact, neither ssleay32 nor Libeay32 exports OPENSSL_sk_num, so I did not modify it.

Reply:
The `BIO_new_ssl_connect` can be both present and not present. I had to do this to make it work:
https://github.com/treeform/quickjwt/blob/master/src/quickjwt/crypto.nim#L36

Please check this with windows its very fragile!

Reply:
In Windows, `BIO_new_ssl_connect` exists only in `ssleay32.dll` or `ssleay64.dll`.

Reply:
In Windows, `BIO_new_ssl_connect` exists only in `ssleay32.dll` or `ssleay64.dll`.